src/@types/common.ts

```diff
@@ -147,6 +147,9 @@ export interface FormControlBaseProps<
 
   /** Make the control readonly */
   readOnly?: boolean;
+
+  /** AutoComplete Content */
+  autoComplete?: string;
```

Comment:
`autoComplete` should not be added to `FormControlBaseProps`, because the props of components such as Rate and DatePicker all extend `FormControlBaseProps`. You should fix it in `AutoCompleteProps`.

Reply:
Done!